interface/orders/orders_results.php

```diff
@@ -766,20 +766,20 @@ function validate(f) {
     if ($reviewauth) {
         ?>
      <center><p>
-         <input type='submit' name='form_submit' value='<?php xl('Sign Results', 'e'); ?>' />
+         <input type='submit' name='form_submit' value='<?php xlt('Sign Results', 'e'); ?>' />
```

Comment:
This is within an html attribute, so use xla. Also need to remove the `e` parameter and add the `echo`